components/02-molecules/menus/main-menu/_00-main-menu.scss

```diff
@@ -15,9 +15,18 @@ $main-menu-medium: $medium;
     left: 0;
     overflow-y: scroll;
     position: absolute;
-    top: 134px;
+    top: 110px;
     right: 0;
     width: 100%;
+
+    @include medium {
+      left: auto;
+      overflow-y: visible;
+      position: relative;
+      top: auto;
+      right: auto;
+      width: auto;
```

Comment:
Let's adjust the ordering of items here to match the rest of the file / be in sync with how we're doing it going forward.